curriculum/challenges/english/25-front-end-development/quiz-git/66f1b06a5a5d10cc100af620.md

```diff
@@ -17,439 +17,442 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Who can use `git`?
 
 #### --distractors--
 
-Placeholder distractor 1
+Developers
 
 ---
 
-Placeholder distractor 2
+Administrators
 
 ---
 
-Placeholder distractor 3
+Subscribers/Premium Users
 
 #### --answer--
 
-Placeholder answer
+Everyone
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is using version control software a best practice for high performing software and DevOps teams?
 
 #### --distractors--
 
-Placeholder distractor 1
+A complete long-term change history of every file
 
 ---
 
-Placeholder distractor 2
+Branching and merging
 
 ---
 
-Placeholder distractor 3
+Traceability
 
 #### --answer--
 
-Placeholder answer
+All of the options
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the maximum storage capacity available for a public repository in GitHub and GitLab respectively?
 
 #### --distractors--
 
-Placeholder distractor 1
+Unlimited, 4GB
 
 ---
 
-Placeholder distractor 2
+5GB, 10GB
 
 ---
 
-Placeholder distractor 3
+5GB, Unlimited
 
 #### --answer--
 
-Placeholder answer
+Unlimited, 5GB
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command would not display the current `git` version?
 
 #### --distractors--
 
-Placeholder distractor 1
+`git --version`
 
 ---
 
-Placeholder distractor 2
+`git -v`
 
 ---
 
-Placeholder distractor 3
+`git version`
 
 #### --answer--
 
-Placeholder answer
+`git v`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Choose the correct option:
+Statement 1 - GPG keys can be used for encrypting data and communication.
+Statement 2 - SSH keys can be used for encrypting data and communication.
```

Comment:
Instead of making kind of two-part question, I'd like more the _Which one is true?_ type question with these statements and mixed _can be used_, _can't be used_. 

This applies also to the other question using similar pattern.